deps/ada.h

```diff
@@ -1957,6 +1957,7 @@ template <typename E>
 #ifdef TL_EXPECTED_EXCEPTIONS_ENABLED
   throw std::forward<E>(e);
 #else
+  (void)e;
```

Comment:
This change should not be necessary. In the worst case, a compiler might issue a warning.

If you believe that there is an issue with this code, it would be best to push a change upstream to https://github.com/TartanLlama/expected

I am sure that @TartanLlama would consider it.

Patching code downstream, without a strong justification, can lead to problems.

I strongly recommend you try to take it upstream first.

Reply:
> In the worst case, a compiler might issue a warning.

Yup, this is purely to suppress a warning.

Reply:
@anonrig see here (just to avoid branching into two conversations). Since this is not a one-liner but rather update of one of Ada's dependencies, perhaps it's best for someone from the team to do that in Ada upstream.

Reply:
I opened a PR to update Ada: https://github.com/ada-url/ada/pull/475

Reply:
So I guess the `(void)e;` is no longer necessary then?

Reply:
> So I guess the (void)e; is no longer necessary then?

Given that @anonrig updated the code in the main ada library, then I think we should just update the binding and this part of the PR would become obsolete.

Short: no, it should not be needed anymore... once @anonrig updates the code here. :-)